"make check-xref" broken: unused local functions

Jenkins complains about the following two unused local function:
```
ERROR: /tmp/test/rabbitmq-public-umbrella/rabbitmq-management/src/rabbit_mgmt_wm_channels_vhost.erl:37 has unused local function(s) rabbit_mgmt_wm_channels_vhost:resource_exists/2
ERROR: /tmp/test/rabbitmq-public-umbrella/rabbitmq-management/src/rabbit_mgmt_wm_connections_vhost.erl:37 has unused local function(s) rabbit_mgmt_wm_connections_vhost:resource_exists/2
make: *** [check-xref] Error 1
```

--- a/src/rabbit_mgmt_wm_channels_vhost.erl
+++ b/src/rabbit_mgmt_wm_channels_vhost.erl
@@ -19,7 +19,7 @@
 %% Lists channels in a vhost
 
 -export([init/1, to_json/2, content_types_provided/2, is_authorized/2,
-         augmented/2]).
+         augmented/2, resource_exists/2]).
 
 -import(rabbit_misc, [pget/2]).
 

--- a/src/rabbit_mgmt_wm_connections_vhost.erl
+++ b/src/rabbit_mgmt_wm_connections_vhost.erl
@@ -19,7 +19,7 @@
 %% Lists connections in a vhost
 
 -export([init/1, to_json/2, content_types_provided/2, is_authorized/2,
-         augmented/2]).
+         augmented/2, resource_exists/2]).
 
 -import(rabbit_misc, [pget/2]).
 

